cmd/test-resources/test-config.yaml

```diff
@@ -2,3 +2,5 @@
 repos:
   - name: test-repo
```

Comment:
have been burned by under-testing the "this is a real Fs not afero in memory" branches a couple times, so continuing  approach in these E2E cmd-level tests of using real file system